### PR TITLE
chore: update overview dashboard for alerts and remove unused chunks dashboards

### PR DESCRIPTION
**Pull request by triclambert**
* Remove unused Cortex/Loki chunks dashboads
* Update overview dashboard
  * Remove total bytes stored panels
  * Remove total storage operations panels
  * Add ring state panels
  * Add query rate panels
  * Add threshold bars for forthcoming alerts
  * Update some panels to have a sort order for the legend

### CODE REVIEW

File: packages/controller/src/resources/monitoring/system/dashboards/opstrace-overview.json [hunk 1]
```diff
@@ -972,18 +977,18 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum by (namespace,code) (rate(grafana_datasource_request_duration_seconds_sum{datasource=\"metrics\"}[5m]) / rate(grafana_datasource_request_duration_seconds_count{datasource=\"metrics\"}[5m]))",
+          "expr": "sum by (job, status_code) (rate(cortex_api_proxy_request_duration_seconds_count{method=\"GET\", route=~\"api_v1_query.*\"}[5m])) OR on() vector(0)",
```
Comment (triclambert) [inline packages/controller/src/resources/monitoring/system/dashboards/opstrace-overview.json:980]: Note: using `OR on() vector(0)` to fill in zeros for instances when there are no queries happening.  This prevents a "no data" panel, which to me usually indicates an error.  Thoughts?

**Comment by sreis** [comment]
@chris did you push all the changes here?
https://system.chris5.opstrace.io/grafana/d/GqFJrOxGk/opstrace-overview-dashboard?orgId=1&refresh=10s

File: packages/controller/src/resources/monitoring/system/dashboards/loki-overview-per-tenant.json [hunk 2]
```diff
@@ -419,7 +419,7 @@
       "steppedLine": true,
       "targets": [
         {
-          "expr": "sum by (route, status_code) (rate(loki_api_proxy_request_duration_seconds_count{job=\"$tenant-tenant.loki-api\", method=\"GET\", route=~\"loki_api_v1_query.*\"}[$timewindow]))",
+          "expr": "sum by (route, status_code) (rate(loki_api_proxy_request_duration_seconds_count{job=\"$tenant-tenant.loki-api\", method=\"GET\", route=~\"loki_api_v1_query.*|loki_api_v1_tail\"}[$timewindow]))",
```
Comment (triclambert) [inline packages/controller/src/resources/monitoring/system/dashboards/loki-overview-per-tenant.json:422]: @jgehrcke this is just a proposal.  What are your thoughts here?

**Comment by triclambert** [comment]
> @chris did you push all the changes here?
> https://system.chris5.opstrace.io/grafana/d/GqFJrOxGk/opstrace-overview-dashboard?orgId=1&refresh=10s

Following-up from Slack:  yes, I did.  We're good to go.  I'll merge this and then we can tweak it tomorrow if necessary.

**Pull request merged**